tests/test_fork_provider.py

```diff
@@ -99,6 +99,16 @@ def test_reset_fork_specify_block_number_via_config(networks, create_fork_provid
     provider.disconnect()
 
 
+@pytest.mark.sync
+def test_set_code(create_fork_provider, fork_contract_instance):
```

Comment:
This is not really related to `fork` mode so this test can be put in `test_provider.py` and use the normal provider.

Reply:
ok great

Reply:
I moved these but theyre failing now.

Reply:
It looks like it was copied and not moved